packages/blockchain-wallet-v4-frontend/src/components/Chart/components/AxisBottom/AxisBottom.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { AxisBottom as VisxAxisBottom } from '@visx/axis'
+
+import { AxisBottomComponent } from './types'
+
+export const AxisBottom: AxisBottomComponent = ({
+  numTicks,
+  scale,
+  textColor,
+  tickFormat,
+  top
+}) => {
+  return (
+    <VisxAxisBottom
+      scale={scale}
+      hideAxisLine
+      hideTicks
+      hideZero
+      numTicks={numTicks}
+      tickFormat={tickFormat}
+      top={top}
+      tickLabelProps={() => ({
+        fill: textColor,
+        fontFamily: 'Inter',
+        fontSize: 14,
+        fontWeight: 600,
+        lineHeight: 22,
```

Comment:
is this a magic number? Could you add it to the top?

Reply:
I removed this component and I am using the AxisBottom from visx in the chart component.
but can I get the lineHeight and size from the app theme?

Reply:
not yet